lib/widgets/crowd_league_app.dart

```diff
@@ -122,3 +123,64 @@ class CheckAuth extends StatelessWidget {
         });
   }
 }
+
+class InitializingIndicator extends StatelessWidget {
```

Comment:
This widget is just a CircularProgressIndicator and some text, in a Material widget so it looks nice.  It's a separate widget as the existing ProgressIndicator widget doesn't need to have it's contents in a Material widget.

Reply:
thanks for the detailed comments :D